File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseTipoAlerta.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.TipoAlertaDTO;

/**
 * Objeto de respuesta con la informacion del tipo de alerta
 * 
 * @author Flor
 *
 */
public class ServerResponseTipoAlerta {

	private List<TipoAlertaDTO> listaTipoAlerta = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseTipoAlerta() {
		super();
	}

	public List<TipoAlertaDTO> getListaTipoAlerta() {
		return listaTipoAlerta;
	}

	public void setListaTipoAlerta(List<TipoAlertaDTO> listaTipoAlerta) {
		this.listaTipoAlerta = listaTipoAlerta;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseTipoAlerta.cs
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseTipoAlerta
    {
        private List<TipoAlertaDTO> _listaTipoAlerta;

        public List<TipoAlertaDTO> listaTipoAlerta
        {
            get { return _listaTipoAlerta; }
            set { _listaTipoAlerta = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
